spec/requests/api/v1/members_spec.rb

```diff
@@ -3,7 +3,131 @@
 require 'rails_helper'
 
 RSpec.describe 'Api::V1::Members', type: :request do
+  before do
+    ENV['JWT_SECRET_KEY'] = 'secret_key'
+  end
+
+  let(:login_user) do
+    create(:user, password: 'password')
+    post api_v1_auth_login_path,
+         params: { user: { email: User.last.email, password: 'password' } }, as: :json
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  let(:valid_headers) { { Authorization: login_user[:token] } }
+
+  let(:valid_attributes) { build(:member) }
+
+  let(:invalid_attributes) { build(:member, name: 123_456) }
+
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    before { create_list :member, 3 }
+
+    it 'renders a successful response' do
+      get api_v1_members_path, headers: valid_headers, as: :json
+      expect(response).to be_successful
+    end
+
+    it 'renders three registers from database' do
+      get api_v1_members_path, headers: valid_headers, as: :json
+      response_hash = JSON.parse(response.body, symbolize_names: true)
+      expect(response_hash[:data].size).to eq(Member.all.size)
+    end
+  end
+
+  describe 'POST /create' do
+    context 'with valid parameters' do
+      it 'creates a new member' do
+        member = create(:member)
+        expect do
+          post api_v1_members_path, params: { name: member.name,
+                                              description: member.description,
+                                              facebook_url: member.facebook_url,
+                                              instagram_url: member.facebook_url,
+                                              linkedin_url: member.facebook_url },
```

Comment:
You could use simply just:
```suggestion
          post api_v1_members_path, params: { valid_attributes },
```